### ISSUE TITLE
Batchnorm gammas are all ones in the released Inception V3 model

### ISSUE THREAD

**Issue by kuprel** [tensorflow/tensorflow#564]
Every batchnorm gamma in the inception V3 model (http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz) is identically ones.  Is this a mistake?

**Comment by vincentvanhoucke** [comment]
No. When using ReLUs (or linear) activations, because they are scale-invariant, the batchnorm gamma is redundant with the scale of the weights of the subsequent convolutional or fully-connected layer, so we don't train them.
